plugin/grpc/README.md

```diff
@@ -31,6 +31,9 @@ Extra knobs are available with an expanded syntax:
 grpc FROM TO... {
     except IGNORED_NAMES...
     tls CERT KEY CA
+    metadata cluster dev
+    metadata clientid dev01
+    metadata pwd xxxx
```

Comment:
Please follow the surrounding usage style/syntax. e.g. a single entry: `metadata KEY VALUE`
Please also order/insert the new option to the end of options instead of inserted between two `tls` related options.

You'll also need to define the new option in the section below, please use like style of the other syntax definitions.